tests/unit/anchore_engine/analyzers/syft/test_syft.py

```diff
@@ -145,3 +129,28 @@ def test_does_not_filter_artifact_by_type(self, pkg_type):
 
         actual = filter_artifacts(artifacts, [])
         assert [a["name"] for a in actual] == ["pkg-name"]
+
+
+@pytest.fixture
+def test_sbom(request):
+    module_path = os.path.dirname(request.module.__file__)
+    test_name = os.path.splitext(os.path.basename(request.module.__file__))[0]
+    with open(
+        os.path.join(
+            module_path, test_name, "{}.json".format(request.node.originalname)
+        )
+    ) as file:
+        return json.load(file)
+
+
+class TestConvertSyftToEngine:
+    @pytest.mark.parametrize(
+        "enable_package_filtering",
+        [False, True],
+    )
+    def test_filter_artifact_by_type(self, test_sbom, enable_package_filtering):
+        findings = convert_syft_to_engine(test_sbom, enable_package_filtering)
+        for pkg_list in findings["package_list"]:
+            assert (
+                "TEST-PACKAGE" not in findings["package_list"][pkg_list]["base"].keys()
```

Comment:
Could you add some checks for packages that are expected to be found?

Reply:
done